root/static/scripts/common/utility/request.js

```diff
@@ -68,4 +68,4 @@ function request(args, context) {
     return promise;
 }
 
-module.exports = request;
+export default request;
```

Comment:
request require in root/static/scripts/common/components/TagEditor.js needs `.default`